src/game.html

```diff
@@ -8,20 +8,45 @@
 	</head>
 
 	<body>
-		<div>
-			<label for="name-label">Enter your name here: </label>
+		<div id="setup-screen">
+			<label for="name-label" data-translate="EnterName">Enter your name here: </label>
```

Comment:
Daca la un element ai pus data-translate, nu mai e nevoie sa scrii si in el text.